src/constants.ts

```diff
@@ -3,19 +3,20 @@ import STAKING_REWARDS_FACTORY_ABI from './abis/staking-rewards-distribution-fac
 import STAKING_REWARDS_DISTRIBUTION_ABI from './abis/staking-rewards-distribution.json'
 import SWPR_CLAIMER_ABI from './abis/swpr-claimer.json'
 import MULTICALL_ABI from './abis/multicall.json'
+import MULTICALL2_ABI from './abis/multicall2.json'
```

Comment:
Pretty sure Multicall2 is backwards compatible with v1, so it should be ok if we only use Multicall2 across the dapp.